crypto/dilithium/pqcrystals_dilithium_ref_common/config.h

```diff
@@ -0,0 +1,37 @@
+#ifndef CONFIG_H
+#define CONFIG_H
+
+#include <openssl/base.h>
+
+//#define DILITHIUM_MODE 2
```

Comment:
?

Reply:
must be vestigial from jake's changes. i'll remove.